assignments/arrays.js

```diff
@@ -63,34 +63,68 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
 
+let car33 = inventory.find((item) => {
+  return item.id === 33
+});
```

Comment:
I love the use of `find`; shortening this with fancy arrow function conveniences would ultimately give you:
```javascript
let car33 = inventory.find(item => item.id === 33);
```